tests/src/test/scala/ch/renku/acceptancetests/ExistingProjectSpec.scala

```diff
@@ -0,0 +1,34 @@
+package ch.renku.acceptancetests
+
+import ch.renku.acceptancetests.model.projects.ProjectDetails
+import ch.renku.acceptancetests.tooling.{AcceptanceSpec, ExistingProjectSpecData}
+import ch.renku.acceptancetests.workflows._
+import ch.renku.acceptancetests.pages._
+
+import scala.language.postfixOps
+
+/**
+  * Run tests on an existing project.
+  */
+class ExistingProjectSpec
+    extends AcceptanceSpec
+    with ExistingProjectSpecData
+    with Collaboration
+    with Login
+    with NewProject
+    with RemoveProject
+    with Settings
+    with Fork {
+
+  scenario("User can fork a project") {
+    implicit val loginType: LoginType = logIntoRenku
+
+    existingProjectDetails match {
+      case Some(projectDetails) => forkTestCase(projectDetails, loginType)
+      case None                 =>
```

Comment:
Is that intentional to skip silently the test if there are no project details available? Actually, in the results of the test, we'll see a green pass for it while effectively nothing was tested.

Reply:
For development or quick verification, it is nice to be able to run a test against an existing project, without the overhead of creating a new project.

I have added some output to explain what happens when no project was provided to run against.